### ISSUE TITLE
[ Feature Request ] Project management

### ISSUE THREAD

**Issue by GBelzoni** [yhat/rodeo#269]
Be great if you could create rodeo "projects" similar to rstudio, so you can create/restore to a project subfolder with all your files, and have it set as the working dir etc.

**Comment by TakenPilot** [comment]
We definitely need this feature, especially if we're going to integrate with git.  I'll add it to the roadmap.

**Comment by GBelzoni** [comment]
Yes! being able to clone projects from git repos would be fantastic.

**Comment by IOpsycode** [comment]
Any updates on this?

In the meantime, what are people using for versioning?
